microsoft-edge/webview2/samples/webview2apissample.md

```diff
@@ -140,14 +160,41 @@ Solution Explorer shows several projects, including the **WebView2APISample** pr
 
 At the top of Visual Studio, set the build target, as follows:
 
-1. In the **Solution Configurations** dropdown list, select **Debug** or **Release**.
+1. In the **Solution Configurations** dropdown list, select **Debug** (or, **Release**).
 
 1. In the **Solution Platforms** dropdown list, select **x86**, **x64**, or **ARM64**.
 
-1. In **Solution Explorer**, right-click the **WebView2APISample** project, and then select **Build**.
+1. In **Solution Explorer**, right-click the **WebView2APISample (Visual Studio 2019)** project, and then select **Build**.
 
    ![The WebView2APISample project selected in Solution Explorer](./webview2apissample-images/webview2apisample-project-selected.png)
 
+   The project is bold, indicating that it's the default in case you use keyboard shortcuts to build.
```

Comment:
@oggy22 @peiche-jessica Do users need to retarget using 2019 build tools?

Reply:
reverted changes in PR other than debug article, will re-apply/ followup in other PR